.ci/bump-release-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
```

Comment:
remove this, after removal of  `getCheckstylePomVersion` .

Reply:
Changes Done